ucsb-cs156-f24/team02-f24-12
# yx - ucsb dining commons menu item create page

Closes #13 Create Page for UCSBDiningCommonsMenuItem

Steps to test: 
1. Click on UCSB Dining Commons Menu Item in the top navigation bar
![image](https://github.com/user-attachments/assets/2b4cc027-1425-492d-934b-402e418c5205)
2. Click on hyperlink "Create"
![image](https://github.com/user-attachments/assets/38da52b8-951e-402a-bc89-53f06de5fdd4)
3. Test with all values and missing values
Correct values:
![image](https://github.com/user-attachments/assets/d63bfcee-3e2f-495c-ad38-21dcb157f081)
Incorrect values:
![image](https://github.com/user-attachments/assets/38573c3f-6515-4666-908c-f52e85f80ad8)
4. Check on Swagger using list all menu items endpoint to check if item is succesfully added
![image](https://github.com/user-attachments/assets/3f9bc4bf-e4f6-4094-a7e5-4ec5e9aea493)
